config/nxf_fastq.config

```diff
@@ -13,6 +13,10 @@ process {
 }
 
 params {
+  minimap2{
+    softClipping = true;
```

Comment:
- softClipping --> soft_clipping
- add doc: `In SAM output, use soft clipping for supplementary alignments (required when STR calling with Straglr)`
- does soft clipping increase have side effects when not using Straglr? increase of cram size? something else?

Reply:
https://medium.com/@lwy730050619/soft-clipping-vs-hard-clipping-in-read-alignment-bd0c96f47426#:~:text=Soft%20clipping%20is%20useful%20for,from%20the%20alignment%20file%20entirely.

The cram files get bigger if I understand this correctly, however this article also suggests that it should be used for primary alignments